src/agent/libclusterfuzz/third-party/crash_analysis/stack_parsing.py

```diff
@@ -0,0 +1,3 @@
+class stack_parser:
```

Comment:
Maybe it will make sense as I read on, but locally and at first glance, this is confusing. Worth a comment, if not a stray commit?